firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/UpdateProgress.java

```diff
@@ -28,21 +28,21 @@
   }
 
   /**
-   * The number of bytes downloaded so far for the APK. Returns -1 if called on an AAB or if no new
-   * release is available.
+   * The number of bytes downloaded so far for an APK. Returns -1 if called when updating to an AAB
+   * or if no new release is available.
    */
   @NonNull
   public abstract long getApkBytesDownloaded();
 
   /**
-   * The file size of the APK file to download in bytes. Returns -1 if called on an AAB or if no new
-   * release is available.
+   * The file size of the APK file to download in bytes. Returns -1 if called when updating to an
+   * AAB or if no new release is available.
```

Comment:
New paragraph for the second sentence, or consider a `@return` clause.

Reply:
done